app/(backend)/dtos/provider.dto.ts

```
@@ -1,128 +1,133 @@
+     */
+    static maskApiKey(apiKey: string): string {
+      return '********';
+    }
+  
+    /**
+     * Mempersiapkan provider untuk respons API dengan data sensitif disamarkan
+     */
+    static toPublicResponse(provider: any): any {
+      return {
+        ...provider,
+        apiKey: this.maskApiKey(provider.apiKey)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)